mmv1/products/alloydb/Cluster.yaml

```diff
@@ -480,3 +481,11 @@ properties:
         description: |
           Name of the primary cluster must be in the format
           'projects/{project}/locations/{location}/clusters/{cluster_id}'
+virtual_fields:
+  - !ruby/object:Api::Type::Enum
+    name: 'deletion_policy'
```

Comment:
We should consider noting that deleting a secondary cluster w/ an instance requires setting this to FORCE, and an error will be returned otherwise.

Reply:
Added note in description